docs/pages/blog/mui-x-v7.md

```diff
@@ -0,0 +1,361 @@
+
+For example, with the `@mui/x-data-grid` npm package, this change led to a reduction of approximately 19% – shrinking the bundle size from [114.2kB](https://bundlephobia.com/package/@mui/x-data-grid@6.19.2) to [92.9kB](https://bundlephobia.com/package/@mui/x-data-grid@7.0.0).
+
+<img width="694" alt="Bundle size change" src="/static/blog/mui-x-v7/new-bundle-size.png" style="padding-left: 20px; padding-right: 5px;">
+
+### New stable features
+
+To ensure that new features can be introduced throughout the lifecycle of a major stable version without waiting for pre-releases, MUI X releases them marked as `experimentalFeatures` or with the `unstable_` prefix.
+
+:::info
+This approach serves as a failsafe in case the API needs to be changed based on user feedback.
+But it's important to note that new features are released **production-ready** regarding behavior and functionality.
+The term 'unstable' refers solely to the potential for breaking changes in minor releases rather than the quality or performance of the features themselves.
+:::
+
+We're excited to announce that the following features have been promoted to stable:
+
+- [Column groups](https://mui.com/x/react-data-grid/column-groups/)
+- [Lazy loading](https://mui.com/x/react-data-grid/row-updates/#lazy-loading) [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+- [Header filters](https://mui.com/x/react-data-grid/filtering/header-filters/) [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+- [Cell selection](https://mui.com/x/react-data-grid/cell-selection/) [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan 'Premium plan')
+- [Clipboard paste](https://mui.com/x/react-data-grid/clipboard/#clipboard-paste) [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan 'Premium plan')
+
+## Tree View
+
+Since the promotion from the lab's package to MUI X, the primary focus has been on improving the component's developer experience.
+This includes efforts aimed to simplify its usage, implement new customization APIs, and provide clearer documentation and illustrative examples, making them more informative and user-friendly.
+
+A significant addition is the time-saving alternative to the traditional JSX architecture, the RichTreeView.
+
+### RichTreeView
+
+We've split the Tree View into two distinct components.
+
+The first is the `SimpleTreeView`, which retains the classic and familiar JSX approach and is still the one behind the original `<TreeView />` tag.
+
+The second is the `RichTreeView`, a new variant designed to streamline the development process, and with which we aim to support several common use cases out-of-the-box.
+
+Similar to the Data Grid, it requires only a dataset and possibly a few properties to render; it does all the heavy lifting for you, handling most of the complexities internally.
```

Comment:
```suggestion
Similar to the Data Grid, it only requires a dataset and possibly a few properties to render; it does all the heavy lifting for you, handling most of the complexities internally.
```

https://english.stackexchange.com/questions/173239/only-requires-or-requires-only


> In the above example, if you wish to emphasize being merely $1, when you would place only as near as possible to $1 and in a prominent position - costs only $1.
>
> There may be times when you would be in the mood of emphasizing that costing $1 is the only consideration you have in mind, and you would choose to say only costs $1.

I think we are closer from the 2nd example here